test/suite/test_compact10.py

```diff
@@ -168,13 +168,12 @@ def test_compact10_incr_backup(self):
         # Take two full backups:
 
         # - The first one will remain untouched.
-        os.mkdir(self.backup_dir_1)
-        self.take_full_backup(self.backup_dir_1)
+        self.setup_directories(self.backup_incr, self.backup_full)
+        self.take_full_backup(self.backup_full)
 
         # - The second one is used for the incremental backups.
-        os.mkdir(self.backup_dir_2)
         self.initial_backup = True
-        self.take_full_backup(self.backup_dir_2)
+        self.take_full_backup(self.backup_incr)
```

Comment:
The naming here makes more sense. We could have dedicated variables for each test.

Reply:
I have changed the naming [3570009](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/3570009210cdd4c711eea7aae6427731f792f903)